### DISCUSSION TITLE
Useful labels for DynAdjust issues?

### DISCUSSION THREAD

**Discussion by jhaasdyk-au** [GeoscienceAustralia/DynAdjust#202] [Q&A] [answered]
As a new contributor to icsm-au/DynAdjust, I have discovered that I have NO rights to add 'Labels' to Issues or Discussions.
- Public members with no special rights in the repo CANNOT add / modify / create labels. 
- Labels (or pre-set New Issues workflows) can assist down-stream triage. 
- Prioritization would still have to be accomplished during triage.

The purpose of this discussion is to identify potential enhancements to the current label set (and or pre-set Issues workflows).
I have raised this here after an initial discussion with Roger Fraser 08 Sept.

t the time of writing, only(?) the default repo labels are available (grouped by me to aid discussion)
- ISSUE TYPE:        bug, enhancement
- ISSUE VALIDITY: invalid, duplicate, 
- OTHER LABELS:  good first issue, help wanted, question 
- PRIORITY:             high priority, nice to have, wont fix

**Comment by jhaasdyk-au** [comment]
EDITED 12/09

I would modify the above and pare it down to:
- ISSUE TYPE: bug(error), improve feature, new feature, docs error, docs update.
- TRIAGE PRIORITY: high priority, medium priority, low priority, nice-not-necessary
- TRIAGE ACTION: pending, assigned, won't fix, invalid, duplicate, help (need input)

Multiple labels might apply.
Alas, I don't know if it is possible to GROUP labels as suggested above... (same colour? shape? other?)

**Comment by rogerfraser** [comment]
Thanks @jhaasdyk-au - these are good suggestions. Simple yet comprehensive.  I'm not sure how to group the labels but we can certainly shade the labels within each category according to a colour range or hue.

**Comment by rogerfraser** [comment, marked as answer]
The [labels](https://github.com/icsm-au/DynAdjust/labels?sort=name-asc) have been reorganised, updated and coloured as per @jhaasdyk-au's suggestions.  Hopefully this addresses the identified shortcomings but please feel free to suggest further improvements.

**Comment by jhaasdyk-au** [comment]
Many thanks @rogerfraser 
I can see those new labels (they look good!).
As before, I cannot add / modify any labelling with my current permissions, and will have to leave that action to whomever does triage.

However, when it comes to creating new ISSUES, 
I find there are two existing workflows  ("Bug Report" and "Feature Request") which automatically add labels. 
This could be one way to expose (some of) these new labels to all users, prior to triage.

![image](https://user-images.githubusercontent.com/81740809/194782519-ecfbd1ee-e54e-4233-a28e-7220b8b3368c.png)
